daemon/logs.go

```diff
@@ -6,5 +6,5 @@ import (
 
 // Logs returns the core's docker service logs.
 func Logs() (io.ReadCloser, error) {
-	return defaultContainer.ServiceLogs(Namespace())
+	return defaultContainer.ServiceLogs([]string{})
```

Comment:
I think container package should be more agnostic. maybe it is better to not access config package inside container. the thing with passing empty _Namespace()_ is kinda weird to me

Reply:
I agree with you, `container.Namespace` function should not access the config to add the Core.Name as prefix. It should be pass the Namespace every time we need it. We could create an helper function in config package to get it simply.

Reply:
what do you think about doing that in another PR / issue. I don't want to make this one overcomplicated. There is already some complicated parts in this PR

Reply:
https://github.com/mesg-foundation/core/pull/478#pullrequestreview-157643597